evals/intent-discovery/live-copilot-harness.eval.ts

```
@@ -0,0 +1,227 @@
+import { existsSync, writeFileSync } from 'node:fs'
```

ESLint

[error] 'node:fs' imported multiple times.

(import/no-duplicates)